### ISSUE TITLE
Liczba sprzedaży produktu z katalogu

### ISSUE THREAD

**Issue by kogol99** [allegro/allegro-api#10109]
Dzień dobry,

W jaki sposób pobrać z API liczbę sprzedanych sztuk lub liczbę osób, które kupiło wybrany produkt?
Wiem, że istnieje /offers/listing i parametr popularity jednak nie odzwierciedla on stanu faktycznego. Na UI Allegro są całkiem inne dane.
Interesuje mnie najlepiej liczba, która pojawia się obok tytułu produktu katalogowego np.: "2 osoby kupiły ostatnio"

**Comment by AureliuszBrussy** [comment]
Nie ma takiej możliwości. Takie dane pobierzesz tylko w imeniu danego sprzedającego (posiadając access token tego użytkownika)  za pomocą [GET /sale/offers](https://developer.allegro.pl/documentation#operation/searchOffersUsingGET).

**Comment by kogol99** [comment]
> Nie ma takiej możliwości. Takie dane pobierzesz tylko w imeniu danego sprzedającego (posiadając access token tego użytkownika) za pomocą [GET /sale/offers](https://developer.allegro.pl/documentation#operation/searchOffersUsingGET).

Czy za pośrednictwem /sale/offers pobiorę liczbę, o której wspomniałem wyżej, w przypadku gdy mam taki produkt wystawiony?

**Comment by AureliuszBrussy** [comment]
Tak, tylko dla wystawionych przez danego sprzedającego ofert (tego, którego posługujesz się access tokenem).

**Comment by kogol99** [comment]
Rozumiem, ale informacje będą wtedy dla produktu katalogowego w ramach całego allegro, czy moja liczba sprzedaży??

**Comment by AureliuszBrussy** [comment]
Tylko liczba Twojej sprzedaży w danej ofercie.

**Comment by kogol99** [comment]
Czy istnieje możliwość dodania takiego parametru do produktu w przyszłości?
Może istnieje możliwość naprawienia popularity w sales/offers, aby była zgodna z UI?

**Comment by AureliuszBrussy** [comment]
Obecnie nie planujemy takiej zmiany. Historycznie zwracaliśmy takie dane, natomiast popularity range to celowa zmiana, o której więcej możesz [przeczytać w newsie](https://developer.allegro.pl/news/get-offers-listing-tylko-dla-zweryfikowanych-aplikacji-GRax4oVgrs1).

**Comment by kogol99** [comment]
Proszę o link do artykuły, o którym wspomniano w artykule, na temat zakresów popularności.

**Comment by AureliuszBrussy** [comment]
To jest wspomniany w artykule news: https://developer.allegro.pl/news/get-offers-listing-nowe-pole-popularityrange-zlq6rWzOluL
